lib/brdocs/changeset.ex

```diff
@@ -0,0 +1,48 @@
+defmodule BrDocs.Changeset do
+  @moduledoc """
+    BrDocs validations to use with `Ecto.Changeset`
+  """
+
+  import Ecto.Changeset
+
+  @doc ~s"""
+  Validates a changeset against `BrDocs` validation rules. Returns an `Ecto.Changeset`.
+
+  Arguments are:
+
+  * `field` - is the changeset field to be validated
+  * `kind` - is the `BrDocs.BrDoc` document kind which will identify the validation rule. It must be one of #{
+    BrDocs.BrDoc.formatted_available_docs()
+  }.
+
+  ## Examples
+
+      defmodule User do
+        use Ecto.Schema
+        import Ecto.Changeset
+        import BrDocs.Changeset
+
+        schema "users" do
+          field :name
+          field :brazilian_doc
+        end
+
+        def changeset(user, params \\\\ %{}) do
```

Comment:
```suggestion
        def changeset(user, params \\ %{}) do
```
Default work with 4 `\`s?

Reply:
I had to escape the doc with `~s` because I had code interpolated. That does seem to break the docs using only 2 `\\`. I had to use 4 `\\\\` to escape and show it ok on docs.

So, that was intentional. (maybe there's a better way of doing that :D)